auth/api/groups/http/endpoint.go

```diff
@@ -194,12 +212,12 @@ func AssignEndpoint(svc groups.Service) endpoint.Endpoint {
 
 func UnassignEndpoint(svc groups.Service) endpoint.Endpoint {
 	return func(ctx context.Context, request interface{}) (interface{}, error) {
-		req := request.(memberGroupReq)
+		req := request.(assignMembersGroupReq)
```

Comment:
`unassignReq` ?

Reply:
it would be the same structure

Reply:
What I mean is that `assignMembersGroupReq` is not consistent with `UnassignEndpoint`